legend-engine-xt-nonrelationalStore-mongodb-executionPlan-test/src/main/resources/core_mongodb_execution_test/test_setup/testSetupDSL.txt

```diff
@@ -0,0 +1,98 @@
+###MongoDB
+Database test::mongodb::mydatabase
```

Comment:
Suppose this can't be a .pure file, as it would try to compile & fail?  (as it doesn't use legend compile?)

Reply:
A pure file won't allow us to add the new "section headers" like ###MongoDB etc